internal/collector/persistentvolume.go

```diff
@@ -108,6 +108,21 @@ var (
 				}
 			}),
 		},
+		{
+			Name: "kube_persistentvolume_capacity",
+			Type: metric.MetricTypeGauge,
+			Help: "persistentvolume capacity in Gigabytes.",
+			GenerateFunc: wrapPersistentVolumeFunc(func(p *v1.PersistentVolume) *metric.Family {
+				storage := p.Spec.Capacity[v1.ResourceStorage]
```

Comment:
could there ever be an additoinal capacity? if so we should make the `v1.ResourceStorage` a label instead, similar to what we do with resource_requests on pod/containers

Reply:
I do not believe there can ever be an additional capacity, because `p.Spec.Capacity` is a `v1.ResourceList`, which in turn is a `map[ResourceName]resource.Quantity`. So, there will only be one `resource.Quantity` for each `ResourceName` in the map (in this case the resource name is `v1.ResourceStorage`.

The name `v1.ResourceList` is a little misleading in that regard, but since it's a map to `resource.Quantity` (which is never a list) I think we're good.